config/loader.go

```diff
@@ -206,6 +207,7 @@ func (s *signalFxLoader) Forwarder(conf *ForwardTo) (protocol.Forwarder, error)
 	if sfConf.TraceSample != nil {
 		sfConf.TraceSample.EtcdServer = conf.Server
 		sfConf.TraceSample.EtcdClient = conf.Client
+		sfConf.TraceSample.AdditionalDimensions = datapoint.AddMaps(conf.TraceSample.AdditionalDimensions, conf.AdditionalDimensions)
```

Comment:
Seems like the precedence order (here and in main.go#L305) should be explicitly documented for the user too.